components/committee/CommitteeInterviewTimes.tsx

```diff
@@ -378,6 +405,30 @@ const CommitteeInterviewTimes = ({
           />
         </div>
       </form>
+
+      {isModalOpen && (
+        <div className="fixed inset-0 z-50 flex items-center justify-center bg-black bg-opacity-50">
+          <div className="flex flex-col bg-gray-100 dark:bg-gray-800 p-5 rounded shadow-lg">
+            <h2 className="mb-4 text-xl font-semibold">
+              Skriv inn navn på rom:
+            </h2>
+            <input
+              type="text"
+              className="my-2 p-2 w-full rounded-lg dark:bg-gray-900  border-gray-900 dark:border-white transition-none outline-none"
+              value={roomInput}
+              onChange={(e) => setRoomInput(e.target.value)}
+            />
+            <div className="flex flex-row justify-between px-5 mt-4">
+              <Button
+                title="Avbryt"
+                onClick={() => setIsModalOpen(false)}
+                color="orange"
+              />
+              <Button title="Ok" onClick={handleRoomSubmit} color="blue" />
+            </div>
+          </div>
+        </div>
+      )}
```

Comment:
Burde kanskje være mulig å trykke enter for å submitte. Tror folk kan synes det er irriterende hvis ikke.

Reply:
Enig🙌

Reply:
Kanskje det hadde vært en idé å beholde forrige verdi i input feltet også? Eller kommer de ikke til å skrive samme rom flere ganger?

Reply:
Tror ikke det er så sannsynlig at det er samme rom, kan bli nesten mer irriterende da å fjerne rom navnet for hver gang